app/modifyWorkshops/WorkshopStep2EventInfo.js

```
@@ -0,0 +1,445 @@
+                  {event.pricingOptions.map((pricing, pricingIndex) => (
+                    <Grid
+                      container
+                      rowSpacing={3}
+                      columnSpacing={2}
+                      sx={{
+                        mb:
+                          pricingIndex !== event.pricingOptions.length - 1
+                            ? 2
+                            : 0,
+                      }}
+                    >
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)